neqo-http3/src/transaction_client.rs

```diff
@@ -14,9 +14,17 @@ use neqo_qpack::encoder::QPackEncoder;
 use neqo_transport::Connection;
 
 use crate::{Error, Res};
+use std::cmp::min;
 use std::mem;
 
-#[derive(Debug)]
+const MAX_DATA_HEADER_SIZE_2: usize = 63; // Maximal amount of data with DATA frame header size 2
+const MAX_DATA_HEADER_SIZE_2_LIMIT: usize = 66; // 63 + 3 (size of the next biffer data frame header)
```

Comment:
```suggestion
const MAX_DATA_HEADER_SIZE_2_LIMIT: usize = 66; // 63 + 3 (size of the next buffer data frame header)
```

But if this is really a sum, then you want to do this:
```suggestion
const MAX_DATA_HEADER_SIZE_2_LIMIT: usize = MAX_DATA_HEADER_SIZE_2 + 3; // 63 + 3 (size of the next buffer data frame header)
```

But then I wonder if this is right.